server/modules/__init__.py

```diff
@@ -0,0 +1 @@
+import server.modules.hash
```

Comment:
Is this file necessary?

Reply:
Its for incase more files are to be add into utils.